src/ccl/.gitignore

```diff
@@ -0,0 +1 @@
+constants.ts
```

Comment:
Ineffective comment: I'm not sure where I think constants.ts should live. Here is probs fine for now.